doc/source/refs.rst

```diff
@@ -21,6 +21,10 @@ References
 .. [BV2004] Boyd, S, and Vandenberghe, L. *Convex optimization*.
    Cambridge university press, 2004.
 
+.. [ComPes2011] Combettes, P.L, and Pesquet, J.-C. *Proximal splitting
```

Comment:
I know this is how it should be, but if you look at the references just below it is also CP, but none of the authors are the same. Are we going to do something about it, or just call this CP2011c?